addons/addon-base-raas/packages/base-raas-services/lib/study/__tests__/study-service.test.js

```diff
@@ -522,126 +602,5 @@ describe('studyService', () => {
         expect(err.message).toEqual('Study with id "doppelganger" does not exist');
       }
     });
-
-    it('should fail due to study having already been updated', async () => {
-      // BUILD
-      const dataIpt = {
-        id: 'doppelganger',
-        rev: 1,
-        accessType: 'readonly',
-      };
-
-      dbService.table.update.mockImplementationOnce(() => {
-        throw error;
-      });
-
-      service.find = jest
-        .fn()
-        .mockImplementationOnce(() => {
-          return { id: 'doppelganger', updatedBy: { username: 'another doppelganger' }, category: 'Organization' };
-        })
-        .mockImplementationOnce(() => {
-          return { id: 'doppelganger', updatedBy: { username: 'another doppelganger' }, category: 'Organization' };
-        });
-
-      // OPERATE
-      try {
-        await service.update({}, dataIpt);
-        expect.hasAssertions();
-      } catch (err) {
-        // CHECK
-        expect(err.message).toEqual(
-          'study information changed just before your request is processed, please try again',
-        );
-      }
-    });
-
-    it('should succeed', async () => {
-      // BUILD
-      const dataIpt = {
-        id: 'doppelganger',
-        rev: 1,
-      };
-      service.find = jest.fn().mockImplementationOnce(() => {
-        return { id: 'doppelganger', category: 'Organization' };
-      });
-      service.audit = jest.fn();
-
-      // OPERATE
-      await service.update({}, dataIpt);
-
-      // CHECK
-      expect(dbService.table.update).toHaveBeenCalled();
-      expect(service.audit).toHaveBeenCalledWith({}, { action: 'update-study', body: undefined });
-    });
-
-    it('should succeed with readwrite accessType', async () => {
```

Comment:
I think there is value in keeping one of these update tests which were changing accessType and assert that we cannot modify accessType any more. Perhaps I missed the test where we were doing this.

Reply:
Good point.  Will do that.